notebooks-auth/test_thredds.ipynb

```diff
@@ -0,0 +1,1019 @@
+{
```

Comment:
<p>This seems to do the same thing as (./optional-components/<a href="https://github.com/bird-house/birdhouse-deploy/tree/master/birdhouse/optional-components/all-public-access" target="_blank">all-public-access</a> + <a href="https://github.com/bird-house/birdhouse-deploy/tree/master/birdhouse/optional-components/secure-thredds" target="_blank">secure-thredds</a>), just  curious why repeat it here.  But no harm I guess.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/PAVICS-e2e-workflow-tests/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Indeed. Repeats that part, but makes readability of the test easier to understand as we explicitly see the permissions defined. </p><p>I found it was very hard to interpret the test when half the permissions were defined externally. </p>